sdk/compute/Azure.ResourceManager.Compute/api/Azure.ResourceManager.Compute.netstandard2.0.cs

```diff
@@ -2347,6 +2349,31 @@ public partial class AdditionalUnattendContent
         public static bool operator !=(Azure.ResourceManager.Compute.Models.AggregatedReplicationState left, Azure.ResourceManager.Compute.Models.AggregatedReplicationState right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public partial class AlternativeOption
```

Comment:
Well the name of this model is too generic. Could we change this to a more descriptive name?

Reply:
do you suggest we update the name in swagger or through directives in autorest.md?

Reply:
We need to add an entry in the `rename-mapping` configuration in the autorest.md file